src/frontend/gatsby/src/components/SearchInput/SearchInput.jsx

```diff
@@ -9,7 +9,7 @@ const useStyles = makeStyles((theme) => ({
     fontSize: '1.6rem',
     '&:hover': {
       border: '2px solid',
```

Comment:
Can we fix this so it uses the same width in the hover and non-hover cases, so it doesn't push the contents if the input when you hover?  That extra 1px moves everything inward vs. just changing the colour.